.goreleaser.yml

```
@@ -1,12 +1,17 @@
 includes:
   - from_url:
-      url: charmbracelet/meta/main/goreleaser.yaml
+      url: charmbracelet/meta/main/goreleaser-full.yaml
 
 variables:
   main: "./cmd/soft"
   binary_name: soft
   description: "A tasty, self-hostable Git server for the command line🍦"
   github_url: "https://github.com/charmbracelet/soft-serve"
-  maintainer: "Christian Rocha <christian@charm.sh>"
-  brew_commit_author_name: "Christian Rocha"
-  brew_commit_author_email: "christian@charm.sh"
+  maintainer: "Ayman Bagabas <ayman@charm.sh>"
+  brew_commit_author_name: "Ayman Bagabas"
+  brew_commit_author_email: "ayman@charm.sh"
+
+before:
+  hooks:
+    - ./scripts/manpages.sh ./cmd/soft soft-serve
+    - ./scripts/completions.sh ./cmd/soft soft-serve
```

this could all go in the meta config:

```suggestion
before:
  hooks:
    - 'go run {{.Var.main}} bash > completions/{{.ProjectName}}.bash'
    - 'go run {{.Var.main}} zsh > completions/{{.ProjectName}}.zsh'
    - 'go run {{.Var.main}} fish > completions/{{.ProjectName}}.fish'
    - sh -c 'go run {{.Var.main}} man | gzip -c >manpages/{{.ProjectName}}.1.gz'
```

something along these lines, maybe with the mkdir first et al